style.css

```diff
@@ -0,0 +1,82 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: "Madimi One", sans-serif;
+    font-weight: 400;
+    font-style: normal;
+    color: burlywood;
+    display: flex;
+    justify-content: center;
+    background-image: url("pancake.jpg");
+    background-position: center;
+    background-size: cover;
+    background-attachment: fixed;
+    background-repeat: no-repeat;
```

Comment:
That's OK, but you can simplify it by using a single `background` shorthand property:
https://developer.mozilla.org/en-US/docs/Web/CSS/background